R/qseaExtra.R

```diff
@@ -7,7 +7,7 @@
 #' @keywords internal
 #' @noRd
 .onLoad <- function(libname, pkgname) {
-  getCGPositions <<- memoise::memoise(getCGPositions)
+  getCGPositions <- memoise::memoise(getCGPositions)
```

Comment:
I think we really do want to memoise this. This caches the result of the function, which is important because it takes a long time to calculate